### ISSUE TITLE
Space between h1 title and paragraph is not consistent between themes

### ISSUE THREAD

**Issue by dpyzo0o** [theme-next/hexo-theme-next#146]
### I agree and want to create new issue 

- [x] Yes, I was on [Hexo Docs page](https://hexo.io/docs/), especially on [Templates](https://hexo.io/docs/templates.html), [Variables](https://hexo.io/docs/variables.html), [Helpers](https://hexo.io/docs/helpers.html) and [Troubleshooting](https://hexo.io/docs/troubleshooting.html).
- [x] Yes, I was on [NexT Documentation Site](http://theme-next.org/docs/).
- [x] And yes, I already searched for current [issues](https://github.com/theme-next/hexo-theme-next/issues?utf8=%E2%9C%93&q=is%3Aissue) and this is not help to me.

***

### Expected behavior 
The space between h1 title and the following paragraph appears to be too small in theme Muse, Pisces and Gemini. But it appears correctly in theme Mist. I wonder if you did these deliberately in different themes or there are some bugs.

In theme Muse, Pisces, Gemini:  
<img width="713" alt="screen shot 2018-02-25 at 20 41 46" src="https://user-images.githubusercontent.com/24768249/36645664-88fd200c-1a6c-11e8-814f-97d0328d27fd.png">

In theme Mist:
<img width="715" alt="screen shot 2018-02-25 at 20 41 07" src="https://user-images.githubusercontent.com/24768249/36645677-d08e3834-1a6c-11e8-8110-bb41eecad32a.png">

Personally I think the way it appears in theme Mist should be the correct way, because normally there should be enough space between title and following paragraph.

### Actual behavior 
As above.

### Steps to reproduce the behavior 
1. N/A
2. N/A
3. N/A

* Link to demo site with this issue: N/A
* Link(s) to source code or any usefull link(s): N/A

### Node.js and NPM Information

```
v9.4.0
5.6.0
```

### Package dependencies Information

```
{
  "name": "hexo-site",
  "version": "0.0.0",
  "private": true,
  "hexo": {
    "version": "3.5.0"
  },
  "dependencies": {
    "hexo": "^3.2.0",
    "hexo-deployer-git": "^0.3.1",
    "hexo-generator-archive": "^0.1.4",
    "hexo-generator-category": "^0.1.3",
    "hexo-generator-index": "^0.2.0",
    "hexo-generator-tag": "^0.2.0",
    "hexo-renderer-ejs": "^0.3.0",
    "hexo-renderer-marked": "^0.3.0",
    "hexo-renderer-stylus": "^0.3.1",
    "hexo-server": "^0.2.0"
  }
}
```

#### Hexo version

```
hexo: 3.5.0
hexo-cli: 1.0.4
os: Darwin 16.7.0 darwin x64
http_parser: 2.7.0
node: 9.4.0
v8: 6.2.414.46-node.17
uv: 1.18.0
zlib: 1.2.11
ares: 1.13.0
modules: 59
nghttp2: 1.29.0
openssl: 1.0.2n
icu: 60.1
unicode: 10.0
cldr: 32.0
tz: 2017c
```

#### Hexo Configuration

```yml
# Hexo Configuration
## Docs: https://hexo.io/docs/configuration.html
## Source: https://github.com/hexojs/hexo/

# Site
title: dpyzo0o's Notes
subtitle:
description: 这个人很懒
author: dpyzo0o
language: zh-CN
timezone:

# URL
## If your site is put in a subdirectory, set url as 'http://yoursite.com/child' and root as '/child/'
url: http://yoursite.com
root: /
permalink: :year/:month/:day/:title/
permalink_defaults:

# Directory
source_dir: source
public_dir: public
tag_dir: tags
archive_dir: archives
category_dir: categories
code_dir: downloads/code
i18n_dir: :lang
skip_render:

# Writing
new_post_name: :title.md # File name of new posts
default_layout: post
titlecase: false # Transform title into titlecase
external_link: true # Open external links in new tab
filename_case: 0
render_drafts: false
post_asset_folder: false
relative_link: false
future: true
highlight:
  enable: true
  line_number: true
  auto_detect: false
  tab_replace:
  
# Home page setting
# path: Root path for your blogs index page. (default = '')
# per_page: Posts displayed per page. (0 = disable pagination)
# order_by: Posts order. (Order by date descending by default)
index_generator:
  path: ''
  per_page: 10
  order_by: -date
  
# Category & Tag
default_category: uncategorized
category_map:
tag_map:

# Date / Time format
## Hexo uses Moment.js to parse and display date
## You can customize the date format as defined in
## http://momentjs.com/docs/#/displaying/format/
date_format: YYYY-MM-DD
time_format: HH:mm:ss

# Pagination
## Set per_page to 0 to disable pagination
per_page: 10
pagination_dir: page

# Extensions
## Plugins: https://hexo.io/plugins/
## Themes: https://hexo.io/themes/
theme: next

# Deployment
## Docs: https://hexo.io/docs/deployment.html
deploy:
  type: git
  repo: git@github.com:dpyzo0o/dpyzo0o.github.io.git
  branch: master

```

### NexT Information

**NexT Version:**

```
[] Latest Master branch.
[x] Latest Release version.
[] Old version -
```

**NexT Scheme:**

```
[x] All schemes
[] Muse
[] Mist
[] Pisces
[] Gemini
```

#### NexT Configuration:

```yml

```

**Comment by sli1989** [comment]
i prefer the margin setting in Mist personally.  😀

**Comment by dpyzo0o** [comment]
But I don't think the author intends to do it this way. For <h2>-<h6> title there are spaces between title and paragraph, it doesn't make sense that <h1> title doesn't have the space. It looks weird if you have both <h1> and other subtitles.

**Comment by dpyzo0o** [comment]
But I don't think the author intends to do it this way. For h2-h6 title there are spaces between title and paragraph, it doesn't make sense that h1 title doesn't have the space. It looks weird if you have both
and other subtitles.

**Comment by ivan-nginx** [comment]
Yes, u are right, @dpyzo0o. Mist was first scheme and it was first custom settings, later all settings became in common style.

**Comment by ivan-nginx** [comment]
Fixed in #148.
